src/chainstate/stacks/mod.rs

```diff
@@ -92,6 +94,7 @@ pub const STACKS_MICROBLOCK_VERSION: u8 = 0;
 pub const MAX_TRANSACTION_LEN: u32 = MAX_MESSAGE_LEN;       // TODO: shrink
 pub const MAX_BLOCK_LEN: u32 = MAX_MESSAGE_LEN;             // TODO: shrink
 
+#[derive(Serialize, Deserialize)]
```

Comment:
This should probably use our `impl_byte_array_serde!` macro -- that macro serializes/deserializes the byte arrays using a hex and then the serde methods for String. It makes the serialization (especially the JSON one) much friendlier.

Reply:
Will do